src/Perpetuum/Commands.cs

```diff
@@ -3473,6 +3473,18 @@ public static Command GetCommandByText(string commandText)
             }
         };
 
+        //GameAdmin Command
+        public static readonly Command EPBonusSet = new Command
```

Comment:
Not anything wrong with this but just a note that you can do the following:

`public static Command EPBonusSet => new Command { ... };`

Which has the added bonus of it now being a read only property, newly instantiated only when needed and destroyed once finished. If CPU cycles are a concern then obviously this isn't a good idea :)